src/plasmapy/simulation/particle_tracker/particle_tracker.py

```diff
@@ -712,7 +712,9 @@ def fract_entered(self):
         The denominator of this fraction is based off the number of tracked
         particles, and therefore does not include stopped or removed particles.
         """
-        return self.num_entered / self.nparticles_tracked
+        return (
```

Comment:
since nparticles_tracked doesn't include removed particles, but num_entered does, this ratio was going >1, giving percentages > 100% in the progress bar. Subtracting the number of removed or stopped particles solves the problem.